src/solvers/dgmulti/types.jl

```diff
@@ -4,32 +4,62 @@
 
 # `DGMulti` refers to both multiple DG types (polynomial/SBP, simplices/quads/hexes) as well as
 # the use of multi-dimensional operators in the solver.
-const DGMulti{NDIMS, ElemType, ApproxType, SurfaceIntegral, VolumeIntegral} =
-  DG{<:RefElemData{NDIMS, ElemType, ApproxType}, Mortar, SurfaceIntegral, VolumeIntegral} where {Mortar}
+const DGMulti{NDIMS, ElemType, ApproxType, SurfaceIntegral, VolumeIntegral} = DG{
+                                                                                 <:RefElemData{
```

Comment:
Do you know why sometimes it adds a newline after `{` and sometimes it doesn't? Here it does...

Reply:
I haven't yet figured out this rule either, sorry 😅